spec/collectionspace/mapper/data_mapper_anthro_spec.rb

```diff
@@ -15,11 +15,15 @@
     end
   
     context 'claim record' do
+      # Tests for claim record are pending because changes must be made to
+      # handling of repeating field groups which contain more than one
+      # field which may be populated by multiple authorities.
+      # Problem in claimantGroupList
```

Comment:
Reason for skipping claim tests for now. I want this in the code base so I can test my refactoring work against it, though.